.github/workflows/loki.yaml

```diff
@@ -29,7 +29,8 @@ on:
       - ConformanceGKE (ci-gke-1.10)
       - ConformanceGKE (ci-gke-1.11)
       - ConformanceGKE (ci-gke-1.12)
-      - ConformanceIngress
```

Comment:
do we still want to keep `ConformanceIngress` for v1.12 branch?

Reply:
ah good call out :100: